src/pages/EditOrderPage.js

```diff
@@ -186,6 +201,14 @@ const EditOrderPage = () => {
       setContact(e.target.value);
     }
   };
+  const validateDeliveryFee = (e) => {
```

Comment:
Some of these functions you can return a boolean then set state in parent component  
  
  
```
  const validateDeliveryFee = (e) => {
      const re = /^[0-9.]+$/;
      return e.target.value === '' || re.test(e.target.value)
  };
```